package.json

```diff
@@ -24,13 +25,15 @@
     "sax": "^1.1.3"
   },
   "devDependencies": {
+    "@types/node": "^7.0.12",
     "assert-diff": "^1.0.1",
     "istanbul": "*",
     "mocha": "*",
     "muk-prop": "^1.0.0",
     "nock": "*",
     "sinon": "^2.0.0",
-    "stream-equal": "~1.0.0"
+    "stream-equal": "~1.0.0",
+    "typescript": "^2.2.2"
```

Comment:
Mocha and istanbul are used directly by this module. But it doesn't use TypeScript directly, it doesn't need it to run tests on travis for example, or when developing for the module.